src/sympc/protocol/falcon/falcon.py

```diff
@@ -451,3 +454,102 @@ def select_shares(x: MPCTensor, y: MPCTensor, b: MPCTensor) -> MPCTensor:
         z = x + (d * (y - x))
 
         return z
+
+    @staticmethod
+    def _random_prime_group(
```

Comment:
This can be extended beyond private compare right? 
Maybe we could have a test for this?

Reply:
Crct ,it could be extended beyond PC compare.
I think we should not test private methods Hrishi
https://softwareengineering.stackexchange.com/questions/100959/how-do-you-unit-test-private-methods/100966
We test it by testing Private Compare.